### DISCUSSION TITLE
Support USDZ on web (not only iOS)

### DISCUSSION THREAD

**Discussion by netgfx** [google/model-viewer#3608] [Ideas]
There is a loader for threejs now https://github.com/mrdoob/three.js/issues/14219 that can support USDZ rendering on the web as well (on Chrome Windows or other platforms/browsers). Perhaps this would be something to consider because USDZ files are generally more compressed than GLB files and often have much smaller sizes which helps the initial load & render.

**Comment by elalish** [comment]
Can you give us some examples of the size difference? That is not our impression; generally speaking their internal data formats are basically equivalent. In fact glTF can get quite a bit more compressed considering it's support for mesh quantization (DRACO) and KTX2 compressed textures that USDZ doesn't have. 

The main reason we don't support USDZ is because they don't have a clear physical spec of their materials like glTF does, which makes it very hard to get consistent rendering between different code bases.
